_pages/en/product-fw-standard.md

```diff
@@ -0,0 +1,404 @@
+
+The Product Owner performs the following activities:
+
+- Acts as a fully integrated team member
+- Uses clear goals to communicate the internal project (release scope) vision at the beginning of every release so that the team understands and is equipped to realize it
+- Makes business decisions and clearly communicates through requirements elaboration techniques (e.g. epics and user stories) so they are understood
+- Works closely with IT Managers in taking the technical aspect of each release scope
+- Ensures compliance with IM/IT-related Treasury Board Directives such as Cyber Security software patching
+- Communicates status outside the team
+
+The above from [Info-Tech Research Group](https://www.infotech.com/research/it-product-owner).
+
+### IT Security Champion Role
+
+The IT Security Champion is the go-to-person for IT Security knowledge about the product.
+
+The IT Security Champion performs the following activities:
+
+- Understands the security tools
+- Implements security tools in the pipeline
+- Analyzes security test results
+- Mitigates identified risks in software-based products
+- Promotes and coaches the team members on security best practices
+
+The above from [SLF's January 2021 presentation](https://014gc.sharepoint.com/:p:/r/sites/DGIIT-Gouvernance-IITB-Governance/SLFFCS/DevSecOps SLF-EN.pptx?d=wc4a2579808df446eab5b3b5b3ceedb5f&csf=1&web=1&e=VVS26e) (modified last point)
+
+### Change Coordinator Role
+
+The Change Coordinator ensures that all changes are managed in a controlled manager, including standard changes and emergency maintenance.
+
+The Change Coordinator performs the following activities:
+
+- Performs impact assessments
+- Seeks the necessary authorisations prior to the change (e.g. IITB Release Process)
+- Ensures all stakeholders are aware of the upcoming product release
+- Documents the change
+
+The above from [Info-Tech Research Group](https://www.infotech.com/research/change-manager).
+
+### Product Support Manager Role
+
+The Product Support Manager is the go to person for any questions or issues relating to the product's support.
+
+The Product Support Manager performs the following activities:
+
+- Establishes the product support process and procedures, both for internal support (within the department) and for external support (Canadians or other departments using the product, if applicable).
+- Is the first point of contact for issue resolution
+- Assesses the priority and severity of issues affecting the product
+- Prioritizes issue resolution based on priority and severity
+- Liaise with IITB's Software Product Specialists, if need be, when COTS software product are being used
+
+### Product Team Organizational Structure
+
+The Product Team Organizational Structure represents the cross-branch Org Chart of the product team.
+
+It identifies which positions, from the ESDC branches involved, that makes up the Product Team.

```

Comment:
I think the 'makes' should be 'make'. And maybe should be "It identifies which positions, from the ESDC branches involved, make up the Product Team"